src/Microsoft.AspNet.SignalR.JS/.npmrc

```diff
@@ -0,0 +1,4 @@
+# Auto generated file from Gardener Plugin CentralFeedServiceAdoptionPlugin
+registry=https://pkgs.dev.azure.com/dnceng/internal/_packaging/internal_PublicPackages/npm/registry/
```

Comment:
?

Reply:
It's a "special" feed that only authorized users can add packages to. Intended to avoid some issues.

Reply:
And, yes, I added everything needed here to the feed